rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java

```diff
@@ -670,17 +670,17 @@ public JobId copyJobAndResubmitWithGeneralInfo(JobId jobId, Map<String, String>
 
     @Override
     public String getCurrentUser() throws NotConnectedException {
-        return ((ISchedulerClient) _getScheduler()).getCurrentUser();
+        return _getScheduler().getCurrentUser();
     }
 
     @Override
     public UserData getCurrentUserData() throws NotConnectedException {
-        return ((ISchedulerClient) _getScheduler()).getCurrentUserData();
+        return _getScheduler().getCurrentUserData();
     }
 
     @Override
     public Map<String, Object> getSchedulerProperties() throws NotConnectedException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)